### DISCUSSION TITLE
Manage language strings in all language files (github)

### DISCUSSION THREAD

**Discussion by rumplin** [louislam/dockge#180] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

I have contributed the translation for Slovenian language in this project.
The project is in it's early stage. 
We can all agree, that there will be more features and things coming into the projects and with this also new language strings.

How will the project maintain what strings have been translated to a language and what still needs translations?

### 🐻 Dockge Version

1.1.1

### 💻 Operating System and Arch

Windows

### 🌐 Browser

Chrome

**Comment by louislam** [comment]
I have a translation platform https://weblate.kuma.pet

But previously it is configured by contributor @401U, I actually do not know how to config it.

@401U Would you like to join my new project? 👀

**Reply by 401U** [reply]
Glad to help,will try at my free time

**Reply by Zaid-maker** [reply]
W 😄

**Comment by Zaid-maker** [comment]
it still says Uptime Kuma in some places!
